.github/workflows/ci-checks.yml

```diff
@@ -0,0 +1,85 @@
+name: CI Checks
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - main
+      - develop
+
+jobs:
+  sensitive-files-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check for sensitive file changes
+        run: |
+          # Get the list of changed files
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }})
+          
+          # Define sensitive files and directories
+          SENSITIVE_PATHS=(
+            ".env"
+            ".env.example"
+            "server/config/keys.js"

```

Comment:
`server/config/keys.js`, `server/config/credentials.json`, `infrastructure/secrets` do not exist in this repository!